### ISSUE TITLE
`*traffic_sign` should not lowercase values

### ISSUE THREAD

**Issue by tordans** [openstreetmap/id-tagging-schema#901]
For the key `traffic_sign` all values are allowed to be upper case, like `DE:123`.

However, for more edge case keys like `cycleway:right:traffic_sign` the value gets transformed to `de:123` by iD.

Is there a way, to apply the "do not lowercase" rule to all keys that follow a pattern like `*:traffic_sign`?
Or is there another feasible workaround?

---

Context https://www.openstreetmap.org/changeset/135924000

**Comment by 1ec5** [comment]
Huh, it shouldn’t be doing that due to:

https://github.com/openstreetmap/id-tagging-schema/blob/8d45c78e651c588fab208c391525c49b98e38644/data/fields/traffic_sign.json#L5-L6

**Comment by 1ec5** [comment]
Oh, I should’ve read more carefully. iD doesn’t have a field for `cycleway:right:traffic_sign`. The automatic lowercasing (or snake-casing) only applies when entering text into a field, but not when editing the raw tags. Maybe iD’s autocompletion feature matched `DE:123` to a `de:123` that already occurs in the database according to taginfo.

**Comment by tyrasd** [comment]
Yep, it's a bug in iD's autocomplete feature. I'll move this issue over to the iD repo
